src/Rings.jl

```diff
@@ -81,6 +81,17 @@ end
 
 *(x::RingElement, y::RingElem) = parent(y)(x)*y
 
+
+"""
+    divexact(x, y)
+
+Return the exact division of `x` by `y`, i.e. an element
```

Comment:
Should be either
```suggestion
Return the result of exact division of `x` by `y`, i.e. an element
```
or
```suggestion
Return the exact quotient of `x` by `y`, i.e. an element
```